### ISSUE TITLE
Much simpler instructions needed

### ISSUE THREAD

**Issue by bradparks** [alexjc/neural-doodle#40]
Hey... This sounds super cool, and I'd love to run it, but find your README.md hard to follow for how to actually run this.

I installed all you suggested, and downloaded VGG19, but wasn't sure how to use it, where to place it, or how to run any example after this. It'd be great if you gave just a few more instructions and a precanned  example on how to go from here.

Thanks!

**Comment by alexjc** [comment]
There are three pre-canned example and install instructions. Did you see them? If so, what's missing?

**Comment by owenman1** [comment]
After doing:
```
python3 -m venv pyvenv --system-site-packages
source pyvenv/bin/activate
python3 -m pip install --ignore-installed -r requirements.txt
```
Put VGG19.pkl.bz2 in the neural-doodle folder and try running:
```
python3 doodle.py --style samples/Monet.jpg --output samples/Coastline.png \
                  --device=cpu --iterations=40
```

**Comment by alexjc** [comment]
That part is already in the README and there's an explicit error message with instructions too. Neither were enough? ;-)

**Comment by bradparks** [comment]
Hey! I don't use Python at all, so maybe that's why i'm missing some stuff - but you're totally right in that you have example files there in the main readme, and included as well.

Some other things I wasn't quite sure of:
- I wasn't sure where to save the VGG19 file, but did put it and the unbzipped file in the repo folder
- I'm not sure exactly where to get numpy and scipy, but googled it and think it looks like [maybe here is a good way on mac](http://stackoverflow.com/a/24616464/26510) or [here](http://www.scipy.org/install.html)?

**Comment by alexjc** [comment]
The approach I use for Mac is `brew` but you can also use Miniconda. Then setup numpy and scipy from there. I'm not sure which is easiest based on some other errors reported, try Miniconda maybe?
